ubuntu20.04/generate-control-file

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+
+function get-arch {
+    if [[ "$HOST_ARCH" =~ ^(broadwell|haswell|skylake|x86_64)$ ]];
```

Comment:
style nit
```
if [ "${a}" =~ ... ]; then
```

Reply:
removed this file to address deb generation as a separate PR